stars/default/name.fab

```diff
@@ -331,7 +359,7 @@
  99171|64_Aql
  99473|65_Aql
  99631|66_Aql
- 99742|67_Aql
+ 99742|67_Aql*
```

Comment:
Why asterisk added here?

Reply:
The punctuation "*" means this star originally in the neighbouring constellation, but it became part of the constellation it is now in with the official laying down of the constellation borders and proper motion.

Reply:
Please remove asterisk from designations, because this file contains modern designations.